### PR TITLE
avro 1.8.2 -> 1.9.2 bump + support for logical types 

### PR DESCRIPTION
**Pull request by arkadius**
(java.time, uuid, decimal)

### CODE REVIEW

File: engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/AvroUtils.scala [hunk 1]
```diff
@@ -1,11 +1,34 @@
 package pl.touk.nussknacker.engine.avro
 
+import org.apache.avro.Conversions.{DecimalConversion, UUIDConversion}
 import org.apache.avro.Schema
+import org.apache.avro.data.TimeConversions
+import org.apache.avro.generic.GenericData
+import org.apache.avro.reflect.ReflectData
+import org.apache.avro.specific.SpecificData
 
 object AvroUtils {
```
Comment (lciolecki) [inline engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/AvroUtils.scala:10]: Can you tell something more about it..?

File: docs/MigrationGuide.md [hunk 2]
```diff
@@ -72,6 +72,8 @@ Additional changes:
 - (Refactor KafkaAvro API) Renamed AvroKeyValueSerializationSchemaFactory to ConfluentAvroKeyValueSerializationSchemaFactory and moved to avro.schemaregistry.confluent package
 - (Refactor KafkaAvro API) Deleted FixedKafkaAvroSourceFactory and FixedKafkaAvroSinkFactory (now we don't support fixed schema)
 
+Be aware that we are using avro 1.9.2 instead of default Flink's 1.8.2 (for java time logical types conversions purpose).
```
Comment (mproch) [inline docs/MigrationGuide.md:75]: Maybe a comment also in build.sbt?

File: engine/flink/avro-util/src/test/scala/pl/touk/nussknacker/engine/avro/schema/PaymentDate.scala [hunk 3]
```diff
@@ -0,0 +1,70 @@
+package pl.touk.nussknacker.engine.avro.schema
+
+import java.time.Instant
+
+import org.apache.avro.generic.GenericData
+
+object PaymentDate extends TestSchemaWithRecord {
```
Comment (mproch) [inline engine/flink/avro-util/src/test/scala/pl/touk/nussknacker/engine/avro/schema/PaymentDate.scala:7]: Can you add more data types to this test?

File: engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/AvroUtils.scala (hunk 1 above)
Comment (lciolecki) [reply engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/AvroUtils.scala:10]: Okey, now I understud it.

**Pull request merged**